### PR TITLE
Local css, blue css

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

**Review by blittle — COMMENTED** [review]
This has the potential to break the following areas in the app:

https://github.com/search?l=&q=org%3ACanopyTax+cpr-multi-selector&ref=advsearch&type=Code&utf8=%E2%9C%93

**Review by blittle — APPROVED** [review]
Looks good. Not that big of a deal, but my only thought is it might be nice to give more context as to what is going on. What does "local css, blue css" mean? The other commit, "removed tests". Why were the tests removed?

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Canopy React Multi-select Component
 + `validate`: function to validate the input. Return true or false.
 + `invalidMsg`: message to display when input is invalidated
 + `doneButton`: add a done button to the dialog when entering input
++ `color`: Color for the pills. Supports blue and green currently.
```
Comment (kentmclean) [inline README.md]: is it that it accepts either of the strings `blue` or `green`?

Comment (kentmclean) [reply README.md]: just a little ambiguous

Comment (kentmclean) [reply README.md]: maybe back-tick those two items to make it more clear

**Review by kentmclean — APPROVED** [review]
👍

**Pull request merged**